tests/data/expression_matrix_bad_missing_keyword.txt

```diff
@@ -0,0 +1 @@
+GE	CELL_0001	CELL_0002	CELL_0003	CELL_0004	CELL_0005	CELL_0006	CELL_0007	CELL_0008	CELL_0009	CELL_00010	CELL_00011	CELL_00012	CELL_00013	CELL_00014	CELL_00015
Itm2a	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
Sergef	0	7.092	0	7.511	6.803	0	0	0	6.783	7.562	0	7.073	6.697	0	5.61
```

Comment:
Same thing with this matrix - it's only one line.  The newlines got removed somehow.

Reply:
b41dfe8